automated/linux/leds/test-leds1.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+source "../test_format.sh"
+
+TEST_NAME="Test LEDs 1 - Tree"
+declare -a LEDs_Core_tree=(
+	"├── a1_green -> ../../devices/platform/leds/leds/a1_green"
+	"├── a1_red -> ../../devices/platform/leds/leds/a1_red"
+	"├── a2_green -> ../../devices/platform/leds/leds/a2_green"
+	"├── a2_red -> ../../devices/platform/leds/leds/a2_red"
+	"├── default-on -> ../../devices/virtual/leds/default-on"
+	"├── mmc0 -> ../../devices/virtual/leds/mmc0"
+	"└── power_red -> ../../devices/platform/leds/leds/power_red"
+)
+
+echo "$TXT_Section1"
```

Comment:
is this echo really needed?  if it is, maybe it is better to put them in a function.

Reply:
It this not needed... 
Maybe with "send-to-lava" it can be deleted... I have not tested it yes...